src/Sylius/Bundle/AddressingBundle/Resources/translations/validators.en.yml

```diff
@@ -29,8 +29,8 @@ sylius:
     country:
         code:
             not_blank: Please enter country ISO code.
-            regex: Country code can only be comprised of letters, numbers, dashes and underscores.
```

Comment:
Last time I've removed translation, there was BC discussion, though I don't see rule for translation at https://docs.sylius.com/en/1.12/book/organization/backward-compatibility-promise.html

I'm in for removal, but lets add a UPGRADE notice.